packages/demux/src/AloxideActionHandler.ts

```diff
@@ -153,6 +224,7 @@ export class AloxideActionHandler extends AbstractActionHandler {
   }
 
   protected async rollbackTo(blockNumber: number): Promise<void> {
-    this.log.debug('-- roll back to block number:', blockNumber);
+    await this.rollbackIndexState(blockNumber);
+    await this.dataAdapter.revert(blockNumber);
```

Comment:
I think if `rollbackIndexState` is executed then we don't need `dataAdapter.revert`.

Reply:
`rollbackIndexState` is roll back data of index state, it update lastProcessedBlockNumber to rollback blockNumber,

`dataAdapter.revert` is revert data of entity such as `poll`, `option`, `vote`